android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -190,6 +190,24 @@ public static boolean firstOnActivityStack()
 	public static void addToActivityStack(Activity activity)
 	{
 		if (activity != null) {
+			if (!isDevelopment && activityStack.size() > 0) {
```

Comment:
I feel like this hack might be a symthome of not using the best possible solution. According to both Apple and Google, apps should have the same core-functionality between dev and prod

Reply:
I only need it for Liveview as that needs the first activity not to be closed to restart the app. But we don't have a way to check for Liveview, that's why I've used dev/prod to do the check

Reply:
Alright, then @prashantsaini1 will know what to do. It's still odd to have this check, but looking at Liveview it may make sense.

Reply:
it works without it but Liveview will look a bit different as you'll see the home screen and then it will launch again.  I don't like the dev/prod difference either :smile: Let's see what Prashant will find :+1: